src/ccalltest.c

```diff
@@ -26,7 +27,7 @@ int __declspec(noinline)
 #else
 int __attribute__((noinline))
 #endif
-testUcharX(unsigned char x) {
+DLLEXPORT testUcharX(unsigned char x) {
```

Comment:
Oh, I thought that since we're setting `CFLAGS` to `JCFLAGS`, we didn't want to pass them.  I'll reinstate `CFLAGS` alongside `CPPFLAGS` then.